Dockerfile

```diff
@@ -28,9 +28,31 @@ RUN     curl -O http://packages.erlang-solutions.com/erlang-solutions_1.0_all.de
 
 # Install erlang
 RUN apt-get update && apt-get install -y \
-        erlang-nox=1:${erlang_version}-1 \
-        erlang-dev=1:${erlang_version}-1 \
-        erlang-src=1:${erlang_version}-1
+        erlang-dev=1:${erlang_version}-* \
```

Comment:
Really isn't there any better solution? :O

Reply:
I followed instructions from the official documentation: https://www.rabbitmq.com/build-server.html#prerequisites

Reply:
I think that it is all about the downloaded and installed package size on the container. `erlang-nox` is smaller than `erlang` (no front-end apps etc.). The easiest solution would be to install `esl-erlang` with awareness that this packages is much bigger.

Reply:
I think we would like to build image as fast as possible. Currently it takes a while.

Reply:
@mentels any comments? I'd would be nice to merge this by the end of the week :)